scripts/jenkins/Makefile.jenkins

```diff
@@ -313,4 +313,12 @@ coverage-junit-algos-jenkins:
 coverage-junit-algos:
 	$(call sed_test_scripts)
 	./gradlew h2o-algos:test $$ADDITIONAL_GRADLE_OPTS
-	./gradlew jacocoMergeExecs $$ADDITIONAL_GRADLE_OPTS
\ No newline at end of file
+	./gradlew jacocoMergeExecs $$ADDITIONAL_GRADLE_OPTS
+
+test-mojo-compatibility:
+ifeq ($(wildcard ../mojoland),)
+	git clone https://github.com/h2oai/mojoland ../mojoland
```

Comment:
I would prefer not to write to the upper-level directory. If "mojoland"  is not found the target should fail and instruct the user to clone it manually.

Jenkins builds should always provide up-to-date checkout in the pipeline itself.

But this solution is fine for now.